redpanda/values.yaml

```diff
@@ -180,7 +227,7 @@ storage:
   # used to store Redpanda's data, otherwise `hostPath` is used.
   persistentVolume:
     # TODO possibly replace with storage.enabled
-    enabled: true
+    enabled: false
```

Comment:
This is a weird line to include in this PR, but it was required to make sure minikube would work. More investigation into why is needed, and there is a ticket in the v2 project related to this: https://github.com/redpanda-data/helm-charts/issues/79